src/shared/tpm2-util.c

```diff
@@ -2062,6 +2062,197 @@ static int tpm2_policy_pcr(
         return tpm2_get_policy_digest(c, session, ret_policy_digest);
 }
 
+/* Extend 'digest' with the PolicyAuthorize calculated hash. */
+int tpm2_calculate_policy_authorize(
+                const TPM2B_PUBLIC *public,
+                const TPM2B_DIGEST *policy_ref,
+                TPM2B_DIGEST *digest) {
+
+        static const TPM2_CC command = TPM2_CC_PolicyAuthorize;
```

Comment:
given this is just an integer, maybe drop the static. i doubt that helps the compiler generate better code, dunno

Reply:
ack